assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterModule.java

```diff
@@ -264,6 +265,7 @@ protected void configure() {
       install(new TracingMetricsModule());
     }
     install(new ExecutorWrapperModule());
+    install(new LogWatcherMetricsModule());
```

Comment:
IMHO, the log watcher would fit much more nicely into `che-core-api-metrics` module where you could better integrate it with other workspace events.

Reply:
done